yaml/src/main/java/com/fasterxml/jackson/jaxrs/yaml/JacksonJaxbYAMLProvider.java

```diff
@@ -0,0 +1,64 @@
+package com.fasterxml.jackson.jaxrs.yaml;
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.fasterxml.jackson.jaxrs.cfg.Annotations;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * JSON content type provider automatically configured to use both Jackson
+ * and JAXB annotations (in that order of priority). Otherwise functionally
+ * same as {@link JacksonYAMLProvider}.
+ *<p>
+ * Typical usage pattern is to just instantiate instance of this
+ * provider for JAX-RS and use as is: this will use both Jackson and
+ * JAXB annotations (with Jackson annotations having priority).
+ *<p>
+ * Note: class annotations are duplicated from super class, since it
+ * is not clear whether JAX-RS implementations are required to
+ * check settings of super-classes. It is important to keep annotations
+ * in sync if changed.
+ */
+@Provider
+@Consumes({"text/yaml", "application/yaml"})
```

Comment:
I assume these are correct, but just wanted to ask if you checked IETF/RFC suggestions for YAML?
Not a big deal as these can be changed easily.